ipad.css

```diff
@@ -0,0 +1,101 @@
+.body {
+    max-width: 768px;
+    margin: 0;
+    padding: 0;
+}
+.header {
+    max-width: 768px;
+}
+.nav__link {
+    display: none;
+}
+.vectors {
+    display: none;
+}
+.nav {
+    margin-bottom: 80.32px;
+}
+
+.nav__input {
+    display: block;
+    width: 35px;
+}
+
+.title {
+    max-width: 370px;
+    margin-bottom: 48px;
+}
+.title__h1 {
+    margin: 0;
+}
+.background {
+    max-width: 768px;
+}
+.image__car {
+    width: 87%;
+}
+
+.box {
+    margin: 0 auto 63px;
+    width: 63%;
+    flex-wrap: wrap;
+}
+
+.box__img-1 {
+    width: 65.92px;
+    height: 50.16px;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Tesla-n/pullRequest?prid=5749910)